src/TimePicker/TimePicker.js

```diff
@@ -190,6 +193,7 @@ TimePicker.defaultProps = {
 
 TimePicker.propDescriptions = {
     ...Time.propDescriptions,
+    popoverProps: 'Additional props to be spread to the [Popover](/popover) component.',
```

Comment:
Do we have a default for this? If not, we should add one so we don't have to rewrite it in all the components.

Reply:
See previous note about using `defaults.js`.